.github/workflows/ci.yml

```
@@ -174,6 +174,28 @@ jobs:
           }
         shell: pwsh
 
+      - name: Validate audio metrics (Unix)
+        if: runner.os != 'Windows'
+        run: |
+          if [ -f artifacts/preview_audio_metrics.json ]; then
+            python -c "import json; d=json.load(open('artifacts/preview_audio_metrics.json'));\
+assert {'integrated_lufs','true_peak_dbfs'} <= d.keys(), 'Missing expected audio metric keys'"
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)